e2e/terraform/packer/linux/setup.sh

```diff
@@ -6,8 +6,8 @@ set -e
 
 # Will be overwritten at test time with the version specified
 NOMADVERSION=0.9.1
-CONSULVERSION=1.7.3
-VAULTVERSION=1.1.1
+CONSULVERSION=1.8.3
+VAULTVERSION=1.5.4
```

Comment:
I had to upgrade the vault version to something recent because the policy in the learn guide requires it.